src/Stryker.Core/Stryker.Core/TestRunners/TestRunResult.cs

```diff
@@ -13,33 +15,36 @@ public TestRunResult(bool success, string message = null)
             Duration = TimeSpan.Zero;
         }
 
-        public TestRunResult(ITestGuids ranTests,
+        public TestRunResult(
+            IEnumerable<VsTestDescription> vsTestDescriptions,
+            ITestGuids ranTests,
```

Comment:
So it's "run-ran-run" thus should be "runTests" in passive voice :D But this would look confusing so I suggest "executed".